terraform/pas_terraform/main.tf

```diff
@@ -0,0 +1,103 @@
+module "project" {
+  source      =  "./modules/project"
+  project     = var.project
+}
+
+module "cloud-credentials" {
+  source                  = "./modules/cloud-credentials"
+  cloud_credentials_name  = var.cloud_credentials_name
+  project                 = var.project
+  client_id               = var.client_id
+  client_secret           = var.client_secret
+  subscription_id         = var.subscription_id
+  tenant_id               = var.tenant_id
+  rolearn                 = var.rolearn
+  externalid              = var.externalid
+  depends_on              = [ module.project]
+}
+
+module "group" {
+  source      = "./modules/group"
+  group       = "${var.project}-project-admin"
+}
+
+module "group-association" {
+  source      = "./modules/group-association"
+  group       = "${var.project}-project-admin"
+  project     = var.project
+  depends_on  = [ module.group]
+}
+
```

Comment:
Remove extra space before all the modules below starting from repositories to eks_cluster.

Reply:
Removed extra spaces before module for a few definitions.